codec-http/src/main/java/io/netty/handler/codec/http/cookie/ClientCookieDecoder.java

```diff
@@ -154,6 +154,7 @@ public Cookie decode(String header) {
         private int expiresEnd;
         private boolean secure;
         private boolean httpOnly;
+        private String sameSite;
```

Comment:
Personally, I would go with an enum.

Reply:
I initially implemented this with an enum in our port of the classes in Styx, but since we have our own classes wrapping netty - and our product is a proxy and not an user agent/server that has to interact with that Cookie - I decided to have an enum there and let Netty passthrough any value.

I am happy to add the attribute as an enum. In that case, what's your preference for error case scenarios? E.g,  if we receive SameSite=invalidValue, should we just throw an IllegalArgumentException?

Reply:
+1 to enum. Wrt illegal values, I'd act as if the attribute isn't present.